contracts/contracts/strategies/BaseCurveAMOStrategy.sol

```diff
@@ -586,6 +593,20 @@ contract BaseCurveAMOStrategy is InitializableAbstractStrategy {
                     Approvals
     ****************************************/
 
+    /**
+     * @notice Sets the maximum slippage allowed for any swap/liquidity operation
+     * @param _maxSlippage Maximum slippage allowed, 1e18 = 100%.
+     */
+    function setMaxSlippage(uint256 _maxSlippage) external onlyGovernor {
+        _setMaxSlippage(_maxSlippage);
+    }
+
+    function _setMaxSlippage(uint256 _maxSlippage) internal {
+        require(_maxSlippage <= 1e18, "Slippage must be less than 100%");
```

Comment:
100% slippage seems extreme. How about a max slippage of 5% (5e16)?
This should protect against the governor miss-configuring the `maxSlippage`